src/server.ts

```diff
@@ -41,10 +42,34 @@ export const SOCKS_PROTOCOLS = ['socks:', 'socks4:', 'socks4a:', 'socks5:', 'soc
 const DEFAULT_AUTH_REALM = 'ProxyChain';
 const DEFAULT_PROXY_SERVER_PORT = 8000;
 
+const HTTPS_DEFAULTS = {
```

Comment:
```suggestion
const HTTPS_DEFAULT_OPTIONS = {
```

(opt) Just more consistent naming across the code...